src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -1133,71 +567,8 @@ private int getNumAltAllelesExcludingNonRef( final VariantContext vc ) {
-     */
-    public static double calculateConcordance( final File actual, final File expected ) {
-        return calculateConcordance(actual, expected, false);
-    }
-
-    /*
-     * Calculate rough concordance between two vcfs, comparing only the positions, alleles, and the first genotype.
-     * Can optionally ignore GVCF blocks in the concordance calculation.
-     */
-    public static double calculateConcordance( final File actual, final File expected, final boolean ignoreGVCFBlocks ) {
-        final Set<String> actualVCFKeys = new HashSet<>();
-        final Set<String> expectedVCFKeys = new HashSet<>();
-        int concordant = 0;
-        int discordant = 0;
-
-        // unzip to avoid https://github.com/broadinstitute/gatk/issues/4224
-        File actualUnzipped = IOUtils.gunzipToTempIfNeeded(actual);
-        try ( final FeatureDataSource<VariantContext> actualSource = new FeatureDataSource<>(actualUnzipped);
-              final FeatureDataSource<VariantContext> expectedSource = new FeatureDataSource<>(expected) ) {
-
-            for ( final VariantContext vc : actualSource ) {
-                if ( ! ignoreGVCFBlocks || ! isGVCFReferenceBlock(vc) ) {
-                    actualVCFKeys.add(keyForVariant(vc));
-                }
-            }
-
-            for ( final VariantContext vc : expectedSource ) {
-                if ( ! ignoreGVCFBlocks || ! isGVCFReferenceBlock(vc) ) {
-                    expectedVCFKeys.add(keyForVariant(vc));
-                }
-            }
-
-            for ( final String vcKey : actualVCFKeys ) {
-                if ( ! expectedVCFKeys.contains(vcKey) ) {
-                    ++discordant;
-                }
-                else {
-                    ++concordant;
-                }
-            }
-
-            for ( final String vcKey : expectedVCFKeys ) {
-                if ( ! actualVCFKeys.contains(vcKey) ) {
-                    ++discordant;
-                }
-            }
-        }
-
-        return (double)concordant / (double)(concordant + discordant);
-    }
-
-    private static String keyForVariant( final VariantContext variant ) {
-        Genotype genotype = variant.getGenotype(0);
-        if (genotype.isPhased()) { // unphase it for comparisons, since we rely on comparing the genotype string below
-            genotype = new GenotypeBuilder(genotype).phased(false).make();
-        }
-        return String.format("%s:%d-%d %s %s", variant.getContig(), variant.getStart(), variant.getEnd(),
-                variant.getAlleles(), genotype.getGenotypeString(false));
-    }
-
-    private static boolean isGVCFReferenceBlock( final VariantContext vc ) {
-        return vc.hasAttribute(VCFConstants.END_KEY) &&
-               vc.getAlternateAlleles().size() == 1 &&
-               vc.getAlternateAllele(0).equals(Allele.NON_REF_ALLELE);
+    @Override
+    public List<String> getToolSpecificArguments() {
+        return Collections.emptyList();
```

Comment:
Maybe make this the default implementation (i.e. in the abstract class) and just have the Spark version override it.